src/segment-layers/run_freesurfer_recon_all.sh

```diff
@@ -19,20 +19,22 @@ echo "Will run recon-all on: "
 echo " - $filename"
 
 # remove the recon-all output otherwise freesurfer complains
-rm -rf $output_dir
-mkdir -p $output_dir
+\rm -rf $output_dir
+\mkdir -p $output_dir
```

Comment:
is the `\` at the beginning of the line a typo or some bah dark magic that I do not know?

Reply:
It escapes any aliasing that users might have made (see; https://github.com/marcobarilari/layerfMRI-toolbox/pull/22#discussion_r2087325401).